File: snippets/ruby/code_sample_5.rb
require "MailchimpMarketing"

mailchimp = MailchimpMarketing::Client.new
mailchimp.set_config ENV["API_KEY"], ENV["API_SERVER"]
list_id = ENV["LIST_ID"]
tag_id = ENV["TAG_ID"]

body = {
  members_to_remove: ["john.smith@example.com", "brad.hudson@example.com"]
}

response = mailchimp.lists.batch_segment_members body, list_id, tag_id

puts "Successfully untagged #{response["total_removed"]} contacts"

File: snippets/node/code_sample_5.js
const listId = "YOUR_LIST_ID";
const tagId = "YOUR_TAG_ID";
const body = {
  members_to_remove: ["john.smith@example.com", "brad.hudson@example.com"]
};

async function run() {
  const response = await apiInstance.batchSegmentMembers(
    body,
    listId,
    tagId
  );

  console.log(`Successfully untagged ${response.total_removed} contacts`);
}

run();